components/dialogs/AuthDialog.vue

```diff
@@ -153,7 +153,10 @@
                     stroke-width="2"
                   />
                 </svg>
-                <div>{{ $t('WalletNoticeDialog.page[1].description[0]') }}</div>
+                <div>
+                  <p>{{ $t('WalletNoticeDialog.page[1].description[0]') }}</p>
+                  <p>{{ $t('WalletNoticeDialog.page[1].description[1]') }}</p>
```

Comment:
Remove the trailing margin from `<p>`
```suggestion
                  <p class="lc-margin-bottom-0">
                    {{ $t('WalletNoticeDialog.page[1].description[1]') }}
                  </p>
```